kroxylicious-operator/pom.xml

```diff
@@ -236,6 +236,11 @@
                     -->
                     <source>src/main/resources/META-INF/fabric8/kafka-proxies.kroxylicious.io-v1.yml</source>
                     <extraAnnotations>true</extraAnnotations>
+                    <existingJavaTypes>
```

Comment:
I not sure what this means? What does it do?  I took a look at fabric8 but didn't find documentation.

Reply:
I get it now